setup.json

```diff
@@ -66,7 +66,7 @@
     },
     "extras_require": {
 	"pre-commit": [
-		"aiida-core[pre-commit] >= 1.2.1,!=1.4.0,!=1.4.1,<2"
+		"aiida-core[pre-commit] >= 2.0.1"
```

Comment:
maybe add <3? or `~2.0.1` for upper bound. There was some discussion yesterday at the webinar for pulling compatibility information in the plugin registry

Reply:
Yes, indeed. `~2.0.1` would only allow `2.0.x` I think, so I am adding `<3`.